app/namespace.go

```diff
@@ -521,6 +533,11 @@ func NewNamespaceCommand(getNamespaceClientFn GetNamespaceClientFn) (CommandOut,
 					Aliases: []string{"rd"},
 					Value:   30,
 				},
+				&cli.StringFlag{
+					Name:  authMethodFlagName,
+					Usage: "The authentication method to use for the namespace (e.g. 'mtls', 'api_key')",
+					Value: AuthMethodMTLS,
```

Comment:
Wonder if we should default this to mTLS and make it non-required, to ensure users don't break scripts when they update tcld.

Reply:
i believe that's what `Value` does :D